### PR TITLE
Preview tally reports

### LINKED ISSUE #246: EM: show a print preview of tally reports
Just like the ballots on screen.

Put that preview behind a button, so that there is a clear intent to show the data.

### PR DESCRIPTION
**Pull request by beausmith**
Resolves #246 

https://user-images.githubusercontent.com/28444/112076843-6abaf980-8b38-11eb-956b-b1a5bbcc64c3.mov

### CODE REVIEW

**Review by carolinemodic — APPROVED** [review]

**Review by eventualbuddha — APPROVED** [review]

File: apps/election-manager/src/App.test.tsx [hunk 1]
```diff
@@ -320,37 +312,6 @@ it('tabulating CVRs with SEMS file', async () => {
...
-
-  const semsRow = domGetByTestId(ballotsByDataSource[0], 'externalvoterecords')
-  domGetByText(semsRow, 'External Results File')
-  domGetByText(semsRow, '100')
-
-  const totalsRow = domGetByTestId(ballotsByDataSource[0], 'total')
-  domGetByText(totalsRow, 'Total')
-  domGetByText(totalsRow, '200')
-
-  const ballotsByVotingMethod = getAllByTestId('voting-method-table')
```
[the remark this answers is not in the document]
Comment (beausmith) [reply apps/election-manager/src/App.test.tsx:337]: 😄  Thanks for catching this! I'm pushing a fix.

File: apps/election-manager/src/screens/TallyReportScreen.tsx [hunk 2]
```diff
@@ -119,64 +129,59 @@ const TallyReportScreen: React.FC = () => {
...
-      votingMethod,
-    }
-  )
 
-  const ballotCountsByVotingMethod = votingMethod
-    ? undefined
-    : outerTallyReport.ballotCountsByVotingMethod
+  useEffect(() => {
+    if (previewReportRef?.current && printReportRef?.current) {
+      previewReportRef.current.innerHTML = printReportRef.current?.innerHTML
```
[the remark this answers is not in the document]
Comment (beausmith) [reply apps/election-manager/src/screens/TallyReportScreen.tsx:140]: Thx. This was a leftover from previous iteration of the logic. Fixing…

File: apps/election-manager/src/screens/TallyReportScreen.tsx [hunk 3]
```diff
@@ -119,64 +129,59 @@ const TallyReportScreen: React.FC = () => {
...
-            }
-            ballotCountsByVotingMethod={ballotCountsByVotingMethod}
           />
           <p>
-            <PrintButton primary>Print {statusPrefix} Tally Report</PrintButton>
+            <PrintButton primary>Print Report</PrintButton>{' '}
+            <Button onPress={toggleReportPreview}>
+              {showPreview ? 'Hide Preview' : 'Preview Report'}
+            </Button>{' '}
+            <Button onPress={handleSaveAsPDF}>Save Report as PDF</Button>
```
[the remark this answers is not in the document]
Comment (beausmith) [reply apps/election-manager/src/screens/TallyReportScreen.tsx:158]: D'oh! Removed. 🤦

File: apps/election-manager/src/screens/TallyReportScreen.tsx [hunk 4]
```diff
@@ -119,64 +129,59 @@ const TallyReportScreen: React.FC = () => {
...
+              <h2>Report Preview</h2>
+              <Text italic small>
+                <strong>Note:</strong> Each report section may be paginated to
+                more than one piece of paper.
+              </Text>
+            </React.Fragment>
+          )}
         </Prose>
+        {showPreview && (
+          <ReportPreview ref={previewReportRef}>preview here</ReportPreview>
```
[the remark this answers is not in the document]
Comment (beausmith) [reply apps/election-manager/src/screens/TallyReportScreen.tsx:179]: Sloppy code! Fixed.

**Pull request merged**